### PR TITLE
Feature 1 is completed

### PR DESCRIPTION
**Pull request by yjash0801**
- feature 1 development is completed
- added user sign up
- added login page

### CODE REVIEW

File: README.MD [hunk 1]
```diff
@@ -1,2 +1,6 @@
 ### GIT
-This is the first commit from master branch
\ No newline at end of file
+This is the first commit from master branch
```
Comment (yjash5668) [inline README.MD:2]: Does this line is required?

**Review by yjash5668 — COMMENTED** [review]
Does this line is required?

File: README.MD (hunk 1 above)
Comment (yjash0801) [reply README.MD:2]: yes, that is required

File: README.MD [hunk 2]
```diff
@@ -1,2 +1,6 @@
...
+
+### feature-1
+* Developer is developing the applications
```
Comment (Adminy801) [inline README.MD:5]: I think we need to change something over here can you do that or is enough

**Review by Adminy801 — COMMENTED** [review]
Line 5

File: README.MD (hunk 2 above)
Comment (yjash0801) [reply README.MD:5]: Firstly, we push if there any problem we can update it later

**Review by Adminy801 — APPROVED** [review]
Approving this PR, good to go

**Pull request merged**